init-project/db-user-schema-create.sql

```diff
@@ -40,6 +43,20 @@ CREATE TABLE `brand` (
   KEY `FK_ce8346e2deaf4543dd3f83304a3` (`businessAccountId`),
   CONSTRAINT `FK_ce8346e2deaf4543dd3f83304a3` FOREIGN KEY (`businessAccountId`) REFERENCES `business_account` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Table structure for table `brand_status`
+--
+
+DROP TABLE IF EXISTS `brand_status`;
+CREATE TABLE `brand_status` (
+  `id` varchar(255) NOT NULL AUTO_INCREMENT,
```

Comment:
AUTO_INCREMENT needs to be removed to be able to execute sql script successfully